client/src/app/queries/sboms.ts

```diff
@@ -116,6 +122,10 @@ export const useDeleteSbomMutation = (
     onSuccess: async (response, id) => {
       onSuccess(response, id);
       await queryClient.invalidateQueries({ queryKey: [SBOMsQueryKey] });
+
+      // Required as SBOMs in other pages might need to to be removed
+      // https://github.com/TanStack/query/discussions/3169#discussioncomment-14347395
```

Comment:
I don't think this comment is needed.
`removeQueries()` goes on par with `invalidateQueries()`, we don't want need to delve into React Queries doco even though it needs to be updated.
If you want to keep it then maybe "`removeQueries()` updates the cache" or equivalent.